geode-core/src/main/java/org/apache/geode/internal/cache/EntryExpiryTask.java

```diff
@@ -253,6 +253,34 @@ protected ExpirationAction getAction() {
     return action;
   }
 
+  @Override
+  protected boolean isIdleExpiredOnOthers() throws EntryNotFoundException {
+    if (getIdleAttributes().getTimeout() <= 0L) {
+      // idle expiration is not being used
+      return true;
+    }
+    if (getIdleAttributes().getAction().isLocal()) {
+      // no need to consult with others if using a local action
+      return true;
+    }
+
+    long latestLastAccessTime = getLatestLastAccessTimeOnOtherMembers();
+    if (latestLastAccessTime > getLastAccessedTime()) {
+      setLastAccessedTime(latestLastAccessTime);
+      return false;
```

Comment:
Is this always true? Can last accessed time from another member can be newer than ours, but still be expired? Do we need to test this new value against the expiration time point?

Reply:
See the new commit that addresses this issue.
If a later lastAccessTime is obtained from others, the caller of this method now rechecks if it is still expired or if it should be rescheduled. This will prevent an extra round of messages in the case in which two members have slightly different lastAccessTimes but are both expired.